spec/lib/krikri/harvesters/oai_harvester_spec.rb

```diff
@@ -229,3 +229,14 @@
     it_behaves_like 'a harvester'
   end
 end
+
+
+describe Krikri::Harvester::Registry do
+  describe '#registered?' do
+    it 'knows OAIHarvester is registered' do
+      # It should have been registered by the engine initializer, engine.rb.
+      expect(described_class.registered?('OAI')).to be true
```

Comment:
See [here](https://github.com/dpla/KriKri/pull/46/files#r22553261) and [here](https://github.com/dpla/KriKri/pull/46/files#r22553388).